visualizer/index.html

```diff
@@ -203,7 +312,7 @@
     #bottomSection {
       display: flex;
       flex: 1;
-      flex-direction: column;
+      overflow-x: auto;
```

Comment:
There's something not quite right here. With a big example, I now see two horizontal scroll bars in the bottom pane.